quizanswers.css

```diff
@@ -0,0 +1,15 @@
+.quiz-question-answer-container{
+    margin: 0rem 2rem;
+}
+.quiz-questions-container{
+    margin-block-start: 0px;
+    margin-block-end: 0px;
+    margin: 0rem 1.7rem;
+    padding-inline-start: 0px;
```

Comment:
Suggestion: using 0 instead of 0rem or 0px will work fine

Reply:
Sarre PR par yehi gadbad karta mai😂

Reply:
Done